### PR TITLE
[LayoutSpecExamples] Added swift counterpart.

### PR DESCRIPTION
**Pull request by george-gw**
ref #2552

### CODE REVIEW

File: examples/LayoutSpecExamples/Sample.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -226,7 +226,7 @@
 			);
 			runOnlyForDeploymentPostprocessing = 0;
 			shellPath = /bin/sh;
-			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [[ $? != 0 ]] ; then\n    cat << EOM\nerror: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\nEOM\n    exit 1\nfi\n";
+			shellScript = "diff \"${PODS_ROOT}/../Podfile.lock\" \"${PODS_ROOT}/Manifest.lock\" > /dev/null\nif [ $? != 0 ] ; then\n    # print error to STDERR\n    echo \"error: The sandbox is not in sync with the Podfile.lock. Run 'pod install' or update your CocoaPods installation.\" >&2\n    exit 1\nfi\n";
```
Comment (maicki) [inline examples/LayoutSpecExamples/Sample.xcodeproj/project.pbxproj]: Do we need to have this change in this commit? I guess you used a different Cocoapods version as we used to create the project?

Comment (george-gw) [inline examples/LayoutSpecExamples/Sample.xcodeproj/project.pbxproj]: Correct, I'm using the latest cocoapods version 1.1.1
I normally revert that line in my commits, but I must have missed it this time. Do you want me to undo this?

Comment (maicki) [inline examples/LayoutSpecExamples/Sample.xcodeproj/project.pbxproj]: Yes would be great! Just undo it and amend it with your latest commit or rebase. Thanks!

Comment (george-gw) [inline examples/LayoutSpecExamples/Sample.xcodeproj/project.pbxproj]: Done :)

**Comment by garrettmoon** [comment]
test this please

**Comment by Adlai-Holler** [comment]
@george-gw We just finished fixing some issues with our CI. Could you rebase when you have a minute?

**Comment by hannahmbanana** [comment]
@george-gw - awesome, thank you!!!

**Comment by george-gw** [comment]
Happy to help :)

**Pull request merged**